community/repo_guidelines.md

```diff
@@ -6,12 +6,12 @@ The ONNX GitHub organization contains a number of repositories. Every repository
 
 ## Rules for all repos
 
-* Must be owned and managed by one of the ONNX SIGs
+* Must be owned and managed by one of the ONNX SIGs or the Steering Committee
 * Must be actively maintained
 * Must adopt the ONNX Code of Conduct
-* Must adopt the standard ONNX license(s)
+* Must adopt the standard ONNX license(s) [All code projects use the Apache 2.0 license. Documentation repositories must use the Creative Commons License version 4.0.]
 * Must adopt the ONNX DCO bot
-* Must adopt all ONNX automation (like LGTM)
+* Must adopt all ONNX automation (like CodeQL)
```

Comment:
Is that really required? Not personally familiar with this.

Reply:
OpenSSF best practices require us to run static code analysis tools. LGTM is now part of GitHub code scanning (https://github.blog/2022-08-15-the-next-step-for-lgtm-com-github-code-scanning/).

Is it not being done in some repos?

Reply:
> Is that really required? Not personally familiar with this. Where does this requirement comes from?

I don't know where this requirements comes from, I just know that LGTM is no longer active since mid-December and onnx/onnx on codeql. 

This is the reason why I would have suggested it for updating.... We could of course also call it static code analysis in general.

Consequently, one must also delete the lgtm.yml file, I add it.

Reply:
let's change it to "static code analysis" for now.  the specific product name or solution may change over time.